docker-compose.yml

```diff
@@ -47,6 +47,7 @@ services:
       STRIPE_PRIVATE_KEY:
       STRIPE_PUBLIC_KEY:
       STRIPE_SIGNING_SECRET:
+      ADMIN_EMAIL:
```

Comment:
This should be documented in the README, i.e. that the user needs to set the `ADMIN_EMAIL` and to what

Reply:
Fixed in 4ac28222.